tests/extract_words_test.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from article.article_class import ArticleClass
+from vec_words.extract_words import add_tokens_to_articles
+from vec_words.token_dict_class import TokenDict
+
+class TestAddTokensToArticles(unittest.TestCase):
+    def test_add_tokens_to_articles_1(self):
+        art_input = ArticleClass('headline', 'publication', 'name')
+        art_input.sub_head = ['test test subhead.']
+        art_input.body_text = ['test word.']
+        tokens = TokenDict()
+        tokens.add('headlin', 1, 1)
+        tokens.add('test', 3, 2)
+        tokens.add('subhead', 1, 2)
+        tokens.add('word', 1, 3)
+        art_output = add_tokens_to_articles([art_input])
+        self.assertDictEqual(art_output[0].tokens, tokens)
+
+    def test_add_tokens_to_articles_2(self):
+        art_input = ArticleClass('test', 'publication', 'name')
+        art_input.sub_head = ['test test test2.']
+        art_input.body_text = ['test test2 test1']
+        tokens = TokenDict()
+        tokens.add('test', 4, 1)
+        tokens.add('test1', 1, 3)
+        tokens.add('test2', 2, 2)
+        art_output = add_tokens_to_articles([art_input])
+        self.assertDictEqual(art_output[0].tokens, tokens)
+
+
+    def test_add_tokens_to_articles_3(self):
+        art_input = ArticleClass('test1', 'publication', 'name')
+        art_input.sub_head = ['test2']
+        art_input.body_text = ['test3']
+        tokens = TokenDict()
+        tokens.add('test1', 1, 1)
+        tokens.add('test2', 1, 2)
+        tokens.add('test3', 1, 3)
+        art_output = add_tokens_to_articles([art_input])
+        self.assertDictEqual(art_output[0].tokens, tokens)
+
+    def test_add_tokens_to_articles_4(self):
```

Comment:
Naming is confusing - maybe name something else than 1, 2, 3, 4?

Reply:
yes